web/resources/views/tool_teacher.blade.php

```diff
@@ -107,10 +107,17 @@
                         <input type="hidden" name="response_id" value="{{$resp->id}}">
                         <button type="submit" class="btn btn-secondary">Update grade</button>
                     </form>@endif
-                </td>
+                </td>   
             </tr>
         @endforeach
     </table>
 
+    <button id="CSV-btn" class="btn btn-primary">Download CSV</button>
```

Comment:
This is basically a link; you click it and it goes somewhere.  We're using Bootstrap 4 as our CSS library here, which allows you to style links like they're buttons: https://getbootstrap.com/docs/4.6/components/buttons/#button-tags

So, removing the button and the script tag, and replacing it with something like:
`<a class="btn btn-primary" role="button" href="/app/exportCSV">Download CSV</a>`
....feels like a cleaner solution here.